doc/security.md

```diff
@@ -68,13 +68,12 @@ Note, however, that those aren't root safe, and a user with root access in such
 More details on container security and the kernel features we use can be found on the
 [LXC security page](https://linuxcontainers.org/lxc/security/).
 
-### Prevent container name leakage
+### Name leakage
```

Comment:
Old heading was correct, as container specific as VMs don't use cgroups.

Reply:
Oh, I wasn't aware of that. But that's great, I was a bit concerned about hiding an instance subsection inside a container-only section. :)